src/org/usfirst/frc/team467/robot/Gyrometer.java

```diff
@@ -156,4 +152,15 @@ public double pidGet() {
 		return angle;
 	}
 
+	@Override
+	public double getAngle() {
+		return getAngleZDegrees();
+	}
+
+	@Override
+	public double getRate() {
+		// TODO Auto-generated method stub
+		return 0;
+	}
+
```

Comment:
I think this file was included in the commit by mistake. It has nothing to do with the Grabber code.